### PR TITLE
ci: Use standard pytest-cov for reporting

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]
let's do it

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Makefile [hunk 1]
```diff
@@ -245,9 +240,9 @@ travis-lint-django-18: travis-lint-postgres
 # Test steps
 travis-test-danger:
 	bundle exec danger
-travis-test-sqlite: test-python-coverage
-travis-test-postgres: test-python-coverage
-travis-test-mysql: test-python-coverage
+travis-test-sqlite: SOUTH_TESTS_MIGRATE=1 test-python
```
Comment (mattrobenolt) [inline Makefile]: This isn't how make works, haha

File: .travis.yml [hunk 2]
```diff
@@ -27,6 +27,7 @@ env:
     - SENTRY_SKIP_BACKEND_VALIDATION=1
     - TRAVIS_NODE_VERSION=4
     - CXX=g++-4.8
+    - SOUTH_TESTS_MIGRATE=1
```
Comment (dcramer) [inline .travis.yml]: not sure if this will have unintended consequences, but seems like an ok idea

**Pull request merged**